app/src/main/res/layout/bottom_left_overlay.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="listener"
+      type="org.oppia.android.app.spotlight.SpotlightNavigationListener" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/overlayConstraintLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/transparent_background"
+    android:clickable="true"
+    android:focusable="true">
```

Comment:
Why should the layout be focusable and clickable? That doesn't seem correct.

Reply:
this makes the screen beneath the overlay unclickable, which is something that we want. See https://github.com/TakuSemba/Spotlight/issues/92